content/docs/pipelines/installation/overview.md

```diff
@@ -0,0 +1,96 @@
++++
+title = "Installation Options for Kubeflow Pipelines"
+description = "Overview of the ways to deploy Kubeflow Pipelines"
+weight = 10
++++
+
+Kubeflow Pipelines offers a few installation options.
+This page describes the options and the features available
+with each option:
+
+* Kubeflow Pipelines as [part of a full Kubeflow deployment](#full-kubeflow).
+* A [standalone](#standalone) Kubeflow Pipelines deployment.
+* **Alpha**: Kubeflow Pipelines deployed [from the
```

Comment:
can we list them in the order of deployment simplicity, i.e. "hosted pipeline", "standalone", "full-kubeflow"?

Reply:
I've moved standalone to the top, so the order is now standalone, full, hosted. I think we should leave hosted as the last option, because (a) it's still in alpha, and (b) it's GCP only. Does that sound good to you too?